collect_app/src/main/java/org/odk/collect/android/adapters/IconMenuListAdapter.java

```diff
@@ -0,0 +1,84 @@
+/*
+ * Copyright 2017 Yura Laguta
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.odk.collect.android.adapters;
+
+import android.content.Context;
+import android.os.Build;
+import android.support.v4.view.ViewCompat;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.BaseAdapter;
+import android.widget.TextView;
+
+import org.odk.collect.android.R;
+import org.odk.collect.android.adapters.model.IconMenuItem;
+
+/**
+ * Adapter for List of options with icons
+ */
+public class IconMenuListAdapter extends BaseAdapter {
+
+    private final Context context;
+    private final IconMenuItem[] items;
```

Comment:
As mentioned before, I think this should be List<IconMenuItem>.

Reply:
fixed.